specification/managementgroups/resource-manager/Microsoft.Management/preview/2024-02-01-preview/management.json

```diff
@@ -0,0 +1,2383 @@
+      "properties": {
+        "properties": {
+          "title": "properties",
+          "$ref": "#/definitions/CreateApplicationRequestProperties"
+        }
+      }
+    },
+    "CreateApplicationRequestProperties": {
+      "description": "Application creation request body parameters.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the application. For example, Running",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The name of the application. For example, ApplicationTest1"
+        },
+        "memberOf": {
+          "title": "memberOf",
+          "$ref": "#/definitions/CreateApplicationMemberOfProperties"
+        }
+      }
+    },
+    "Application": {
+      "description": "The application details.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource application."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "properties": {
+          "title": "properties",
+          "$ref": "#/definitions/ApplicationProperties"
+        }
+      }
+    },
+    "ApplicationProperties": {
+      "description": "Application creation request body parameters.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the application. For example, Running",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The name of the application. For example, ApplicationTest1"
+        },
+        "memberOf": {
+          "title": "memberOf",
+          "$ref": "#/definitions/CreateApplicationMemberOfProperties"
+        }
+      }
+    },
+    "CreateApplicationMemberOfProperties": {
+      "description": "The details of the parent management group or application.",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

Don't duplicate top-level resource envelope property names within the property bag. Recommend "resourceId".